### DISCUSSION TITLE
[Question] How to remove "Also on" and Disqus comment section

### DISCUSSION THREAD

**Discussion by smholvoet** [dotnet/docfx#8272] [General]
I'm wondering how I can either remove or completely hide the "Also on" and Disqus comment sections at the bottom of each page:

![image](https://user-images.githubusercontent.com/13507392/84125822-6a37f300-aa3d-11ea-8abb-de7e12b0445b.png)

These get added to the page by default upon creating a project. I've tried searching in the `partials` of the default template, but couldn't really find anything suitable.

**Comment by KalleOlaviNiemitalo** [comment]
Which version of DocFX is this, and how do you create the project?

<https://github.com/dotnet/docfx/pull/1663> added disqus to `Documentation/template/`, which <https://github.com/dotnet/docfx/pull/2461> moved to `Documentation/custom_template/`, but that is not the default template.

**Comment by smholvoet** [comment]
@KalleOlaviNiemitalo 

I've used version `2.54.0.0`.

Project which I've used is the [Documentation](https://github.com/dotnet/docfx/tree/master/Documentation) folder of this project. I've tried using that as a starting point.

**Comment by KalleOlaviNiemitalo** [comment]
OK, removing `"custom_template"` from here should do the trick: <https://github.com/dotnet/docfx/blob/19762d9ef69fefc73cd5282e377b5a4319081d80/Documentation/docfx.json#L66-L69>

And then you can also remove the `custom_template` directory.

**Comment by KalleOlaviNiemitalo** [comment]
`docfx init` is intended as the starting point, I think. [Quickstart](<https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-as-a-command-line-tool> "Use DocFX as a command-line tool | Getting Started with DocFX | DocFX website")
